core/src/main/scala/kafka/log/LogCleaner.scala

```diff
@@ -110,8 +110,8 @@ class LogCleaner(initialConfig: CleanerConfig,
                                         "bytes",
                                         time = time)
 
-  /* the threads */
-  private val cleaners = mutable.ArrayBuffer[CleanerThread]()
+  // visible for testing
```

Comment:
nit: can you remove these? They are a little distracting and I think the fact that we have package access defined is good enough to indicate reduced scope.